src/index.js

```diff
@@ -5,10 +5,8 @@
 const request = require("request"),
   path = require("path"),
   tar = require("tar"),
-  zlib = require("zlib"),
```

Comment:
Newer version of `tar` already uses `zlib` under the hood if the file is a gzipped one